sk0-front/src/app/home/home.page.html

```diff
@@ -1,13 +1,3 @@
-<ion-header [translucent]="true">
-  <ion-toolbar>
-    <ion-title>home</ion-title>
-  </ion-toolbar>
-</ion-header>
-
 <ion-content [fullscreen]="true">
-  <ion-header collapse="condense">
-    <ion-toolbar>
-      <ion-title size="large">home</ion-title>
-    </ion-toolbar>
-  </ion-header>
+  <h1 class="ion-text-center">Üdvözöllek kedves <br />{{userName}}!</h1>
```

Comment:
Styling should not be achieved by content, we should rather create different blocks and/or flexboxes.

Reply:
Szerintem ezen az oldalon egyelőre csak annyi lesz az érték, ha elkészül az authService, hozzá lehessen férni a névhez, később/nemsokára ez kuka lesz. "Styling should not be achieved by content" ? - ezt nem értem

Reply:
Az uj sort inkabb css-sel kellene odarakni, minthogy html-lel.

Reply:
kicsit módosítottam rajta

Reply:
módosítottam a következő commitben